<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_user</class>
 <widget class="QDialog" name="dialog_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuración de Usuario</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>338</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="User_verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="User_Config_verticalLayout">
      <item>
       <widget class="QLabel" name="User_Config_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Configuración de Usuario y Contraseña</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_ID_User">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ID_User_Num_User">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="User_Config_User_horizontalLayout">
        <item>
         <widget class="QLabel" name="User_Config_User_label">
          <property name="text">
           <string>Usuario:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="User_Config_User_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="User_Config_Pass_horizontallayout">
        <item>
         <widget class="QLabel" name="User_Config_Pass_label">
          <property name="text">
           <string>Contraseña:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="User_Config_Pass_lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="USer_Config_Word_horizontalLayout">
        <item>
         <widget class="QLabel" name="User_Config_label_Word">
          <property name="text">
           <string>Contraseña Secreta:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="User_Config_lineEdit_Word">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="User_Config_checkBox">
        <property name="text">
         <string>Habilitar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="User_Pref_verticalLayout">
      <item>
       <widget class="QLabel" name="User_Pref_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Preferencias</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="User_Pref_Temp_horizontalLayout">
        <item>
         <widget class="QLabel" name="User_Pref_Temp_label">
          <property name="text">
           <string>Temperatura:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="User_Pref_Temp_spinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="User_Pref_Luz_horizontalLayout">
        <item>
         <widget class="QLabel" name="User_Pref_Luz_label">
          <property name="text">
           <string>Luz:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="User_Pref_Luz_spinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>6</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="User_Pref_checkBox">
        <property name="text">
         <string>Habilitar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="User_Data_verticalLayout">
      <item>
       <widget class="QLabel" name="User_Data_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Datos Actuales</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="User_Data_label_Temp">
        <property name="text">
         <string>Temperatura</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="User_Data_lcdNumber_Temp">
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="User_Data_label_Luz">
        <property name="text">
         <string>Luz</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="User_Data_lcdNumber_Luz"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="User_Serie_label">
        <property name="text">
         <string>Conexión al dispositivo. Listado de los Puertos disponibles:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="User_Serie_comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="User_Serie_pushButton">
        <property name="text">
         <string>Conectar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="User_Serie_label_Con">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="User_pushButton">
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
